### PR TITLE
Add todo app

### PR DESCRIPTION
**Pull request by tintinthong**
Handled in other PRs
- multi select 
- multi select integration
- kanban 
- kanban integration

### CODE REVIEW

File: packages/realm-server/migrations/1726671342065_backfill-realm-owners.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ exports.up = (pgm) => {
          VALUES
            ('https://realms-staging.stack.cards/seed/', '@seed_realm:stack.cards', true, true, true),
            ('https://realms-staging.stack.cards/experiments/', '@experiments_realm:stack.cards', true, true, true),
-           ('https://realms-staging.stack.cards/experiments/', '*', true, false, false),
+           ('https://realms-staging.stack.cards/experiments/', '*', true, true, false),
```
Comment (tintinthong) [inline packages/realm-server/migrations/1726671342065_backfill-realm-owners.js]: Whoops. Do not commit this tintinthong

File: packages/boxel-ui/addon/src/components/avatar/index.gts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  if (!string) {
+    return 'transparent';
+  }
+
+  let hash = 0;
+  let i;
+
+  for (i = 0; i < string.length; i += 1) {
+    hash = string.charCodeAt(i) + ((hash << 5) - hash);
+  }
```
Comment (richardhjtan) [inline packages/boxel-ui/addon/src/components/avatar/index.gts]: If the `i` is used inside the loop, perhaps we can declare inside the loop

for (let i = 0; i < string.length; i += 1) {

File: packages/boxel-ui/addon/src/components/avatar/index.gts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+  let color = '#';
+
+  for (i = 0; i < 3; i += 1) {
```
Comment (richardhjtan) [inline packages/boxel-ui/addon/src/components/avatar/index.gts]: Same thought as line 12

File: packages/boxel-ui/addon/src/components/avatar/index.gts [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    const value = (hash >> (i * 8)) & 0xff;
+    color += `00${value.toString(16)}`.substr(-2);
```
Comment (richardhjtan) [inline packages/boxel-ui/addon/src/components/avatar/index.gts]: `substr` is deprecated: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr

It is encouraged to replace `substr` to `slice` on modern approach: https://stackoverflow.com/questions/2243824/what-is-the-difference-between-string-slice-and-string-substring

File: packages/boxel-ui/addon/src/components/avatar/index.gts [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+  }
+
+  return color;
+}
+
+interface Signature {
+  Args: {
+    // CSS length value
```
Comment (richardhjtan) [inline packages/boxel-ui/addon/src/components/avatar/index.gts]: Do we need these comments?

File: packages/boxel-ui/addon/src/components/radio-input/index.gts [hunk 6]
```diff
@@ -16,9 +17,10 @@ export interface Signature {
     hideRadio?: boolean;
     invalid?: boolean;
     items: any[];
+    keyName?: string;
     name: string;
     orientation?: string;
-    spacing?: string;
+    spacing?: string; // New argument to specify the key name
```
Comment (richardhjtan) [inline packages/boxel-ui/addon/src/components/radio-input/index.gts]: @tintinthong not sure if this comment is necessary

File: packages/experiments-realm/app-task-card.gts [hunk 7]
```diff
@@ -0,0 +1,561 @@
...
+    return this.getQueryForStatus('In Review');
+  }
+
+  get stagedQuery() {
+    return this.getQueryForStatus('Staged');
+  }
+
+  get shippedQuery() {
+    return this.getQueryForStatus('Shipped');
+  }
```
Comment (richardhjtan) [inline packages/experiments-realm/app-task-card.gts]: @tintinthong is that a chance we can shorten the code to as a helper to receive dynamic key instead

eg: getQueryByStatus(key)

File: packages/experiments-realm/app-task-card.gts [hunk 8]
```diff
@@ -0,0 +1,561 @@
...
+
+  columnData: ColumnData[] = [
+    {
+      status: {
+        label: 'Backlog',
+        index: 0,
+      },
+      query: this.backlogQuery,
```
Comment (richardhjtan) [inline packages/experiments-realm/app-task-card.gts]: then eg  `this.getQueryByStatus('Backlog')`

File: packages/realm-server/migrations/1726671342065_backfill-realm-owners.js (hunk 1 above)
Comment (lucaslyl) [reply packages/realm-server/migrations/1726671342065_backfill-realm-owners.js]: The new update from the main branch has already fixed this, so we don't need to edit this file to modify the experiment card.

**Review by richardhjtan — APPROVED** [review]
LGTM!

**Pull request merged**